app/src/main/res/values/strings.xml

```diff
@@ -285,19 +285,29 @@ Please report any problems you encounter via Github.</string>
     <string name="category_debug">Debug</string>
     <string name="category_experimental">Experimental</string>
 
-    <string name="always_run_in_background">Always run in background</string>
+    <!-- Preference screen - Run conditions -->
+    <string name="always_run_in_background">to_be_deleted</string>
+    <string name="always_run_in_background_enabled">to_be_deleted</string>
+    <string name="always_run_in_background_disabled">to_be_deleted</string>
+    <string name="sync_only_charging">to_be_deleted</string>
+    <string name="sync_only_wifi">to_be_deleted</string>
+    <string name="sync_only_wifi_ssids">to_be_deleted</string>
 
-    <!-- Preference summary in case it is enabled -->
-    <string name="always_run_in_background_enabled">Syncthing always runs in the background, according to preferences below.</string>
+    <string name="run_conditions_title">Run Conditions</string>
+    <string name="run_conditions_summary">Use the following options to decide when Syncthing will run.</string>
 
-    <!-- Preference summary in case it is disabled -->
-    <string name="always_run_in_background_disabled">Syncthing only runs when explicitly started, and can be stopped by menu button.</string>
+    <string name="run_on_mobile_data_title">Run on mobile data</string>
+    <string name="run_on_mobile_data_summary">Run when device is connected via the mobile data network. Warning: This can consume a lot of data from your mobile operator data plan if you sync large amounts of data.</string>
 
-    <string name="sync_only_charging">Run only when charging</string>
+    <string name="run_on_wifi_title">Run on wifi</string>
+    <string name="run_on_wifi_summary">Run when device is connected to a wifi network.</string>
 
-    <string name="sync_only_wifi">Run only on wifi</string>
+    <string name="run_on_metered_wifi_title">Run on metered wifi</string>
+    <string name="run_on_metered_wifi_summary">Run when device is connected to a metered wifi network e.g. a hotspot or tethered network. Attention: This can consume large portion of your data plan if you sync a lot of data.</string>
 
-    <string name="sync_only_wifi_ssids">Restrict to certain wifi networks</string>
+    <string name="run_on_whitelisted_wifi_title">Run on specified wifi networks</string>
+    <string name="run_on_whitelisted_wifi_networks">Run on selected wifi networks: %1$s</string>
```

Comment:
Maybe "Run on selected wifi networks **only**" or "Run **only** on selected wifi networks".  
(Bold is just for purposes of review, not to be done in the actual UI).